vars/buildPlugin.groovy

```diff
@@ -102,14 +94,11 @@ def call(Map params = [:]) {
                                 artifacts = '**/build/libs/*.hpi,**/build/libs/*.jpi'
                             }
 
-                            timestamps {
-                                junit testReports
-                                if (failFast && currentBuild.result == 'UNSTABLE') {
-                                    error 'There were test failures; halting early'
-                                }
-                                archiveArtifacts artifacts: artifacts,
-                                            fingerprint: true
+                            junit testReports // TODO do this in a finally-block so we capture all test results even if one branch aborts early
```

Comment:
Not feeling confident enough to do that refactoring without ability to test the effect.